### PR TITLE
release 2.1.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,10 +26,6 @@ These should not impact the functionality of the module.
 - Add OMAPI key configuration [\#90](https://github.com/voxpupuli/puppet-dhcp/issues/90)
 - Add define for specifying DHCP classes [\#86](https://github.com/voxpupuli/puppet-dhcp/issues/86)
 
-**Breaking changes:**
```
Comment (ekohl) [inline CHANGELOG.md]: Could you leave this in?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+[Full Changelog](https://github.com/voxpupuli/puppet-dhcp/compare/v2.0.0...v2.1.0)
+
+**Implemented enhancements:**
+
+- Update $extra\_config to work with arrays and not only strings [\#156](https://github.com/voxpupuli/puppet-dhcp/pull/156) ([bastelfreak](https://github.com/bastelfreak))
+- Support SmartOS [\#155](https://github.com/voxpupuli/puppet-dhcp/pull/155) ([petems](https://github.com/petems))
+
+**Merged pull requests:**
+
+- release 2.0.1 [\#153](https://github.com/voxpupuli/puppet-dhcp/pull/153) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (ekohl) [inline CHANGELOG.md]: This tag was never pushed to github and the forge?

File: CHANGELOG.md (hunk 1 above)
Comment (bastelfreak) [reply CHANGELOG.md]: oh :(
That got lost somehow

File: CHANGELOG.md (hunk 2 above)
Comment (bastelfreak) [reply CHANGELOG.md]: oh :(
That got lost somehow

File: CHANGELOG.md (hunk 1 above)
Comment (bastelfreak) [reply CHANGELOG.md]: regenerated the changelog. looks better now

File: CHANGELOG.md (hunk 2 above)
Comment (TraGicCode) [reply CHANGELOG.md]: Can we rectify this beforre merging this PR?  I assume we need an admin to cut a release manually to the forge.

Comment (ekohl) [reply CHANGELOG.md]: I don't see which feature was actually added. Maybe the PR should be renamed to 'Prepare 2.0.1'.

Comment (bastelfreak) [reply CHANGELOG.md]: @ekohl IMO we should release this as 2.1.0, we added two new features. I'm not sure if 2.0.1 was supposed to be released, but I don't think so.

Comment (ekohl) [reply CHANGELOG.md]: @bastelfreak that's what I meant. If we renamed the #153 to ```Prepare 2.0.1``` then the changelog matches reality and all would be well.

Comment (bastelfreak) [reply CHANGELOG.md]: ah got you, will do

**Pull request merged**